PathToTina Documentation (#1181)
tinacms/tinacms.org: content/docs/tina-cloud/faq.md

We will contact you if we believe your use case may eventually fit within our post-beta paid plans.

## **Does Tina Cloud work with Monorepos?**

> It does! Tina Cloud can work with sites inside monorepos by configuring the path to your .tina folder in your Projects.

Reviewer: ```suggestion
It does! Tina Cloud can work with sites inside monorepos by specifying the path to your `.tina` folder in your Project configuration.
```

> If your repo isn't a monorepo, then there's no need to do any configuration. We'll look for your Tina Config at the root of your repo.

Reviewer: ```suggestion
If your repo is not a monorepo, there's no need to do any configuration. We'll expect your `.tina` folder at the root of your repo.
```

Here's an example monorepo structure that works with Tina Cloud:

    /projects/site-a
    /projects/site-b

>     /projects/site-c

Reviewer: format as code [block](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks)?

Author: fixed
